### PR TITLE
Add 868 MHz Ukrainian Band

### PR DESCRIPTION
**Pull request by Krezalis**
## Thank you for sending in a pull request, here's some tips to get started!

(Please delete all these tips and replace with your text)

- Before starting on some new big chunk of code, it it is optional but highly recommended to open an issue first
  to say "hey, I think this idea X should be implemented and I'm starting work on it. My general plan is Y, any feedback
  is appreciated." This will allow other devs to potentially save you time by not accidentially duplicating work etc...
- Please do not check in files that don't have real changes
- Please do not reformat lines that you didn't have to change the code on
- We recommend using the [Visual Studio Code](https://platformio.org/install/ide?install=vscode) editor and the 'clang-format' extension,
  because automatically follows our indentation rules and it's auto reformatting will not cause spurious changes to lines.
- If your PR fixes a bug, mention "fixes #bugnum" somewhere in your pull request description.
- If your other co-developers have comments on your PR please tweak as needed.

### CODE REVIEW

File: src/mesh/RadioInterface.cpp [hunk 1]
```diff
@@ -105,6 +105,12 @@ const RegionInfo regions[] = {
         This needs to be last. Same as US.
     */
     RDEF(UNSET, 902.0f, 928.0f, 100, 0, 30, true, false, false)
+        
+    /*
+        868,0-868,6 Mhz 25 mW
+        https://nkrzi.gov.ua/images/upload/256/5810/PDF_UUZ_19_01_2016.pdf
+    */
+    RDEF(UA868, 868.1f, 868.5f, 100, 0, 20, true, false, false)
```
Comment (caveman99) [inline src/mesh/RadioInterface.cpp:113]: Please add the name of the definition to https://github.com/meshtastic/protobufs/blob/e00b5ba7d06053d820f1123351881fc4fa9270d1/config.proto#L506 or else you won't be able to use it. Also please add an underscore in front of the frequency designator to come in line with the other definitions.

Suggestion: UA_868

**Review by caveman99 — CHANGES_REQUESTED** [review]

**Comment by caveman99** [comment]
Also: if you don't have another lora range in your country (except the worldwide Lora_24) you can omit the 868 and simply call the region UA

File: src/mesh/RadioInterface.cpp [hunk 2]
```diff
@@ -105,6 +105,12 @@ const RegionInfo regions[] = {
         This needs to be last. Same as US.
     */
     RDEF(UNSET, 902.0f, 928.0f, 100, 0, 30, true, false, false)
+
```
[the remark this answers is not in the document]
Comment (Krezalis) [reply src/mesh/RadioInterface.cpp:108]: Current regulation https://nkrzi.gov.ua/images/upload/256/5810/PDF_UUZ_19_01_2016.pdf

**Comment by Krezalis** [comment]
https://github.com/meshtastic/protobufs/pull/271

File: src/mesh/RadioInterface.cpp [hunk 3]
```diff
@@ -100,12 +100,24 @@ const RegionInfo regions[] = {
     */
 
     RDEF(LORA_24, 2400.0f, 2483.5f, 100, 0, 10, true, false, true),
-
+    
+    /*
+        433,05-434,7 Mhz 10 mW
+        https://nkrzi.gov.ua/images/upload/256/5810/PDF_UUZ_19_01_2016.pdf
+    */
+    RDEF(UA_433, 433.0f, 434.0f, 100, 0, 12, true, false, false),
```
Comment (caveman99) [inline src/mesh/RadioInterface.cpp:108]: Page 121 states for РІ 42-2: "Робочий цикл менше 10 %" (Duty Cacle not more than 10%) This makes it the same definition as EU_433 (ETSI) and can be omitted.

File: src/mesh/RadioInterface.cpp [hunk 4]
```diff
@@ -100,12 +100,24 @@ const RegionInfo regions[] = {
...
+    
+    /*
+        868,0-868,6 Mhz 25 mW
+        https://nkrzi.gov.ua/images/upload/256/5810/PDF_UUZ_19_01_2016.pdf
+    */
+    RDEF(UA_868, 868.1f, 868.5f, 100, 0, 14, true, false, false),
+
```
Comment (caveman99) [inline src/mesh/RadioInterface.cpp]: Sorry, but this is not correct. "Робочий цикл менше 1 %. Робочий цикл може бути більше 1% за умови
використання режиму контролю спектра LBT (режим прослуховування перед
включенням передавача) та широкосмуговою модуляцією з розширенням
спектру методом стрибкоподібної зміни частоти" 

We are not employing frequency hopping or spread spectrum techniques, although we use LBT. So this allocation is restricted to 1% duty cycle.

(P.S. you can define the true borders of the spectrum and don't need to keep a guard distance, the firmware takes care of that. so you can really say it's for 868,0f to 868,6f

Comment (caveman99) [reply src/mesh/RadioInterface.cpp]: You may be better off using the EU_868 allocation, it's harmonized by ETSI to use 10% instead of 1% duty cycle.

**Review by caveman99 — CHANGES_REQUESTED** [review]

File: src/mesh/RadioInterface.cpp (hunk 3 above)
Comment (Krezalis) [reply src/mesh/RadioInterface.cpp]: extended freq to 434.7

File: src/mesh/RadioInterface.cpp (hunk 4 above)
Comment (Krezalis) [reply src/mesh/RadioInterface.cpp]: Hm, та = & so if it use only LBT its not be 100% its bad. Ok, for official build it set 1%.

**Review by caveman99 — APPROVED** [review]

**Pull request merged**